### PR TITLE
add divider scene

### PR DESCRIPTION
**Pull request by annieyw**
First draft:
image

### CODE REVIEW

File: scenes/src/app/scenes/divider/divider-scene.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
+app-divider-scene .mat-list .mat-list-item {
+  font-size: 40px;
+  height: 130px;
```
Comment (mmalerba) [inline scenes/src/app/scenes/divider/divider-scene.scss:3]: No need to mess with the dimensions on everything, just use `transform: scale()` (Does the scene viewer have built in support for this? I remember discussing it

**Comment by annieyw** [comment]
image

File: src/app/shared/stack-blitz/stack-blitz-writer.spec.ts [hunk 2]
```diff
@@ -60,7 +60,7 @@ describe('StackBlitzWriter', () => {
       `method="post" target="_blank"></form>`);
   });
 
-  it('should add files to form input', () => {
+  it('should add files to form divider', () => {
```
Comment (mmalerba) [inline src/app/shared/stack-blitz/stack-blitz-writer.spec.ts:63]: renaming mistake?

File: src/app/shared/stack-blitz/stack-blitz-writer.ts [hunk 3]
```diff
@@ -125,7 +125,7 @@ export class StackBlitzWriter {
     return form;
   }
 
-  /** Appends the name and value as an input to the form. */
+  /** Appends the name and value as an divider to the form. */
```
Comment (mmalerba) [inline src/app/shared/stack-blitz/stack-blitz-writer.ts:128]: ?

**Pull request merged**